ACKImagePicker/ViewControllers/ACKImagePicker.swift

```diff
@@ -28,6 +28,7 @@ open class ACKImagePicker: UINavigationController {
     convenience init() {
         let rootController = ImagePickerViewController()
         self.init(rootViewController: rootController)
+        navigationBar.isTranslucent = false
```

Comment:
if this is a problem only on iOS 10 then I would prefer to set it only on iOS 10. What do you think?

Reply:
I don't think that UI of this component (at least the translucency) is such dealbreaker that it is worth an extra `if` 🤷